yield_plugin/config/parameters.yaml

```diff
@@ -11,7 +11,7 @@ on_route_vehicle_collision_horizon_in_s: 10.0
 # Minimum speed for moving obstacle
 # Units: m/s
 # Value type: Desired
-min_obstacle_speed_in_ms: 0.5
+min_obstacle_speed_in_ms: 0.25
```

Comment:
I reduced this because when I tried to use this value, it seemed bit unrealistic. 0.25m/s is more acceptable